Plugins/NativeCamera/Include/Babylon/Plugins/NativeCamera.h

```diff
@@ -7,6 +7,11 @@ namespace Babylon::Plugins
     class Camera final
     {
     public:
+        struct CameraDimensions {
+            uint32_t width;
+            uint32_t height;
+        };
```

Comment:
Shouldn't this be in NativeCameraImpl.h? This is not part of the public contract, just the internal one, yes?

Reply:
I put it here to not redefine it for every .h file, but I can move it.

Reply:
I would move it if it is not intended to be part of the public API. As mentioned in my other comment, we need to rework the impl headers and stuff to reduce duplication, but we could do that later.